docs/manual/forms/faq.rst

```diff
@@ -0,0 +1,50 @@
+.. _faq:
+
+==========================
+Vaak voorkomende problemen
+==========================
+
+Advanced logica regels
+======================
+
+Selectievakjes
+--------------
+
+Selectievakjes in FormIO hebben een 'nested structure'. Bijvoorbeeld, als er een selectievakjes component met key
+``selectBoxes`` is en mogelijke waarden ``a``, ``b`` en ``c``, dan heeft de data de structuur:
+
+.. code-block:: json
+
+    {
+        "a": false,
+        "b": false,
+        "c": false
+    }
+
+Als de ``a`` vakje is aangevinkt, heeft de inzending data de volgende structuur:
```

Comment:
```suggestion
Als het ``a``-vakje is aangevinkt, heeft de inzending data de volgende structuur:
```

Reply:
Arghhh I should know always het +je!! :roll_eyes: